src/Client/Utils.fs

```diff
@@ -1,13 +1,15 @@
 module Client.Utils
 
 open Fable.Import
+open Fable.Core.JsInterop
 
-let load<'T> key =
+let inline load<'T> key =
     Browser.localStorage.getItem(key) |> unbox
-    |> Option.map (JS.JSON.parse >> unbox<'T>)
+    |> Option.map ofJson<'T>
 
-let save key (data: 'T) =
-    Browser.localStorage.setItem(key, JS.JSON.stringify data)
+let inline delete key =
+    Browser.localStorage.removeItem(key)
 
-let delete key =
-    Browser.localStorage.removeItem(key)
\ No newline at end of file
+let inline save<'T> key (data: 'T) =
+    delete key
```

Comment:
Why `delete key`? `Browser.localStorage.setItem` will just overwrite if I'm not wrong.

Reply:
Yes that was an attempt to fix a bug that I had. Not needed at all.